airlock/notifications.py

```diff
@@ -0,0 +1,33 @@
+import requests
+from django.conf import settings
+
+
+session = requests.Session()
+
+
+def send_notification_event(event_json: str, username: str):
+    """API call to job server to create a release."""
+    if not settings.AIRLOCK_API_TOKEN:
+        # Skip attempting to send notifications when we're running a
+        # local dev server in isolation
+        return {"status": "ok"}
```

Comment:
Is it worth logging these (assuming that the logs appear in the console) so that it's easy to see what would have been sent when running locally?